packages/components/src/internal/components/forms/QueryInfoQuantity.spec.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { Input } from 'formsy-react-components';
+import { QueryInfoQuantity } from "./QueryInfoQuantity";
+import { mount, ReactWrapper } from "enzyme";
+import { RadioGroupInput } from "./input/RadioGroupInput";
+import { DERIVATIVE_CREATION, POOLED_SAMPLE_CREATION } from "../../../index";
+import Formsy from 'formsy-react';
+
+describe("<QueryInfoQuantity>", () => {
+    function validate(wrapper: ReactWrapper, optionCount: number, includeCount: boolean ) {
+        expect(wrapper.find(RadioGroupInput)).toHaveLength(optionCount === 0 ? 0 : 1);
+        expect(wrapper.find(Input)).toHaveLength((includeCount || optionCount > 0) ? 1 : 0);
+    }
+
+    test("no content", () => {
+        const wrapper = mount(
+            <QueryInfoQuantity
+                creationTypeOptions={undefined}
+                includeCountField={false}
+                maxCount={5}
+                countText={"Quantity"}
+            />
+        );
+        validate(wrapper, 0, false);
+        wrapper.unmount();
+    });
+
+    test("no options, show quantity", () => {
+        const wrapper = mount(
+            <Formsy>
```

Comment:
we have a couple other test helpers for things like "wrap a component for server context" or "wrap a component for draggable", we should keep this in mind for Formsy in case this comes up more and we need to move this to a test helper.